ansible/host_vars/localhost

```diff
@@ -0,0 +1,16 @@
+---
+# FIXME(w-miller): set this more cleverly?
+base_path: "~/tenks/"
+allocations_file_path: "{{ base_path + '/allocations.yml' }}"
+
+flavours:
```

Comment:
We tend to resign ourselves to the Americanisation `flavors`, given to do otherwise in this case is a syntax error.

Reply:
The `flavours` here refers to a Tenks *flavour* (a set of specs from which to create a bare metal node), rather than a Nova *flavor*. Since Tenks and Nova flavo(u)rs are conceptually similar, I had thought I'd give them similar names, using the 'u' to distinguish between them; but come to think of it that's probably quite confusing.

I guess it's probably best to think of a different name for this concept in Tenks.

Reply:
Great plan.  And I'm sure in verbal discussion they are completely distinct from one another and not confusing at all :-)